views/index.hbs

```diff
@@ -174,6 +174,8 @@
 <script src="https://unpkg.com/packery@2/dist/packery.pkgd.min.js"></script>
 <!-- draggabilly CDN -->
 <script src="https://unpkg.com/draggabilly@2/dist/draggabilly.pkgd.min.js"></script>
+<!-- IE Polyfills -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.6.15/browser-polyfill.min.js"></script>
```

Comment:
could you move this into the ie block?

```
<!--[if IE]>
  <!-- IE Polyfills -->
   <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.6.15/browser-polyfill.min.js"></script>
<![endif]-->
```

And it'd go better in dependencies. These are dependencies specifically for index, put polyfills should be included for everything.